languages/go/exercises/concept/maps/.docs/instructions.md

```diff
@@ -0,0 +1,46 @@
+A friend of you has an old wholesale store called **Gross Store**, the name came from the quantity of the item that the store sell, it's all in [gross unit][gross-unit]. Your friend asked you to implement a point of sale (POS) system for his store, **but first, you want to build a prototype for it, in your prototype, your system will only record the quantity**. Your friend gave you a list of measurements to help you:
+
+| Unit               | Score |
+| ------------------ | ----- |
+| quarter_of_a_dozen | 3     |
+| half_of_a_dozen    | 6     |
+| dozen              | 12    |
+| small_gross        | 120   |
+| gross              | 144   |
+| great_gross        | 1728  |
+
+## 1. Store the unit of measurement in your program
+
+In order to use the measurement, you need to store the measurement in your program.
```

Comment:
Could you add examples for all the tasks? See [this example](https://github.com/exercism/v3/blob/master/languages/go/exercises/concept/numbers/.docs/instructions.md#1-calculate-the-production-rate-per-hour) how to do that.

Reply:
15c7949 commit added example, but I leave out the example for task 1, since the example is the actual answer, is that ok? By the way, what's the purpose of an example in each task?

Reply:
Yeah sure. You don't want to give away the solution.

> the purpose of an example in each task?

The purpose is for students to get a concrete example on what the function will do. This is especially useful for students working in-browser, as they won't have access to the tests. For more information, see [this video](https://www.youtube.com/watch?v=gkbBqd7hPrA&t=309)